YoutubeDLSharp.Tests/DownloadTests.cs

```diff
@@ -19,8 +19,9 @@ public static async Task Initialize(TestContext context)
         {
             await PrepTests.DownloadBinaries();
             ydl = new YoutubeDL();
-            ydl.YoutubeDLPath = "yt-dlp.exe";
-            ydl.FFmpegPath = "ffmpeg.exe";
+            ydl.YoutubeDLPath = ydl.YoutubeDLPath;
+            ydl.FFmpegPath = ydl.FFmpegPath;
+            ydl.OutputFileTemplate = "%(title)s.%(ext)s";
```

Comment:
The output file template needs to stay or the assertions for the filenames will need to change, as I changed the default file template to match what yt-dlp used (`%(title)s [%(id)s].%(ext)s`). Otherwise, the assertions would fail, as the video title would end up being something like "TEST VIDEO C0DPdy98e4c" instead of "TEST VIDEO"